staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -754,6 +752,24 @@ func (s *store) ttlOpts(ctx context.Context, ttl int64) ([]clientv3.OpOption, er
 	return []clientv3.OpOption{clientv3.WithLease(id)}, nil
 }
 
+// ensureMinimumResourceVersion returns an error unless provided actualRevision from etcd is greater
```

Comment:
Will it be easier to follow the comment if it is written in positive sense? Something like:

ensureMinimumResourceVersion returns an error if provided actualRevision from etcd is less than the minimumResourceVersion.

Reply:
That is better, thanks! Fixed.